tests/test_main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import pytest
+
+from app.main import is_isogram
+
+
+@pytest.mark.parametrize(
+    "word, expected",
+    [
+        ("abcdefghijklmnopqrstuvwxyz", True),
+        ("ABCdeFghijKlmnOpqrsTuvwxyZ", True),
+        ("ZYXWVUTSRQPONMLKJIHGFEDCBA", True),
+    ]
+)
+def test_for_all_alphabet_in_mixed_case(word, expected):
+    assert is_isogram(word) is expected
+
+
+def test_for_empty_string():
+    assert is_isogram('') is True
+
+
+@pytest.mark.parametrize(
+    "word, expected",
+    [
+        ("FhjhuigfffFfFfFfFfFfFfAaAaAa", False),
+        ("haroldddddddddddd", False),
+        ("AEZakmiohfodsfhudafidafbudiafbdui", False)
+    ]
+)
```

Comment:
You are testing both cases here: repeated letters and repeated case insensitive letters. Either make separate test for this or use `pytest.param`

Reply:
I've fixed